catalog/catalog.yaml

```diff
@@ -339,3 +344,34 @@ components:
       type: object
       additionalProperties:
         $ref: '#/components/schemas/ProviderInfo'
+
+    SubscribeConfig:
```

Comment:
Can we rename this SubscribeOpts? Since everything else in the ProviderInfo ends with `Opts`